packages/integration/src/lib/workspace/workspace.state.ts

```diff
@@ -1,16 +1,29 @@
-import { Injectable } from '@angular/core';
+import { Injectable, OnDestroy } from '@angular/core';
 
-import { BehaviorSubject } from 'rxjs';
+import { BehaviorSubject, Subscription } from 'rxjs';
 
-import { EntityRecord, Workspace, WorkspaceStore } from '@igo2/common';
+import { EntityRecord, Workspace, WorkspaceStore, Widget } from '@igo2/common';
 
 /**
  * Service that holds the state of the workspace module
  */
 @Injectable({
   providedIn: 'root'
 })
-export class WorkspaceState {
+export class WorkspaceState implements OnDestroy {
+
+  public workspacePanelExpanded: boolean = false;
+
+  readonly workspaceEnabled$: BehaviorSubject<boolean> = new BehaviorSubject<boolean>(false);
+
+  /** Subscription to the active workspace */
+  private activeWorkspace$$: Subscription;
+
+  /** Subscription to the active workspace widget */
+  private activeWorkspaceWidget$$: Subscription;
+
+  /** Active widget observable. Only one may be active for all clients */
```

Comment:
The comment about "clients" is not applicable here.

Reply:
Fixed